index.js

```diff
@@ -0,0 +1,38 @@
+(function() {
+
+  const Actuate = animations => $ => new Promise ((resolve, reject) => {
+
+    const commands = Array.isArray(animations) ?
+      animations : animations.split(' ')
+
+    const event = [
+      'animationsend',
+      'webkitAnimationEnd',
+      'oAnimationEnd',
```

Comment:
```function whichTransitionEvent(){
    var t;
    var el = document.createElement('fakeelement'); // you could probs just reference `document.body` for this
    var transitions = {
      'transition':'transitionend',
      'OTransition':'oTransitionEnd',
      'MozTransition':'transitionend',
      'WebkitTransition':'webkitTransitionEnd'
    }

    for(t in transitions){
        if( el.style[t] !== undefined ){
            return transitions[t];
        }
    }
}```
obvs changign transition to animationend, its from modernizr. Those events wont be there if the browser doesnt support em so it means a user on edge would get `animationEnd` but a user on ie 9 would get `-ms` etc etc